### PR TITLE
chore(core): fix log severity on postmark calls

### PR DESCRIPTION
**Pull request by progsmile**
#4669

### CODE REVIEW

File: apps/core/lib/core/email/postmark.ex [hunk 1]
```diff
@@ -15,12 +15,8 @@ defmodule Core.Email.Postmark do
 
       {:ok, bounce_id}
     else
-      {:error, error} = error_result ->
-        Logger.error("[PostmarkClient]: #{error}")
-        error_result
-
       error ->
-        Logger.error("[PostmarkClient]: error happened #{inspect(error)}")
+        Logger.info("[PostmarkClient]: #{inspect(error)}")
```
Comment (BatrakM) [inline apps/core/lib/core/email/postmark.ex]: So now you will have {:error, error} tuple in logs? Or {:error, error} path never matched?

Comment (progsmile) [reply apps/core/lib/core/email/postmark.ex]: Yes, I've decided not to match error result, only for log message separation. Do you think it's worth to do?

Comment (BatrakM) [reply apps/core/lib/core/email/postmark.ex]: Well, that's not a lot of code to keep for a more clean error log, but on the other side 10 extra symbols in log message is not a bit deal. Let's hear some other opinions maybe

Comment (AlexKovalevych) [reply apps/core/lib/core/email/postmark.ex]: why not warn?

Comment (progsmile) [reply apps/core/lib/core/email/postmark.ex]: yep, warn is better

Comment (progsmile) [reply apps/core/lib/core/email/postmark.ex]: Because we can't do nothing regarding that

Comment (progsmile) [reply apps/core/lib/core/email/postmark.ex]: Agree?

**Pull request merged**